lib/elixir/lib/inspect/algebra.ex

```diff
@@ -285,13 +287,13 @@ defmodule Inspect.Algebra do
       ["a", "b", "c"]
 
   """
-  @spec concat([t]) :: doc_cons
-  def concat(docs) do
+  @spec concat([t]) :: t
+  def concat(docs) when is_list(docs) do
     fold_doc(docs, &concat(&1, &2))
   end
 
   @doc ~S"""
-  Nests document entity `x` positions deep.
+  Nests the given document `positions` positions deep.
```

Comment:
I think it should be `position` (singular), "positions" should be a list of integers